scully.rxjs-docs.config.ts

```diff
@@ -1,12 +1,12 @@
 import { ScullyConfig, setPluginConfig } from '@scullyio/scully';
 import { baseHrefRewrite } from '@scullyio/scully-plugin-base-href-rewrite';
 
-setPluginConfig(baseHrefRewrite, { href: '/rxjs-docu/' });
+/* setPluginConfig(baseHrefRewrite, { href: '/rxjs-docu/' }); */
 export const config: Partial<ScullyConfig> = {
   projectRoot: './src',
   projectName: 'rxjs-docs',
   outDir: './dist/static',
-  defaultPostRenderers: [baseHrefRewrite],
+  /* defaultPostRenderers: [baseHrefRewrite], */
```

Comment:
This comment shouldn't be applied :-)

Reply:
Sh*t, that should not have been committed. My mistake. Fixing it now.